keel/api/v1/leads/views.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from keel.api.permissions import CustomLeadPermission
+from rest_framework import status
+from .serializers import CustomerLeadSerializer
+
+class CustomerLeadView(APIView):
+
+    # permission_classes = (CustomLeadPermission, )
+
+    def post(self, request, format="json"):
+        serializer = CustomerLeadSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        
+        valid_data = serializer.validated_data
```

Comment:
have you saved the lead object...
there is no serializer.save() statement anywhere.....

Reply:
Oh, an oversight, fixed :)

Reply:
bro , please test the API using postman before you raise PR.

Reply:
Actually I did, but I didnt check the admin to see if it's saved, didn't check db either. Sorry about that